__tests__/app.test.js

```diff
@@ -166,9 +167,8 @@ describe("app", () => {
         .post("/api/articles/1/comments")
         .send(inputBody)
         .expect(201)
-        .then((response) => {
-          const comment = { comment: response.body.comment[0] };
-          expect(comment.comment).toEqual(
+        .then(({ body: { comment } }) => {
+          expect(comment).toEqual(
             expect.objectContaining({
               comment_id: expect.any(Number),
```

Comment:
you can be exact with these expect statements, as you know the precise properties you want back here